### DISCUSSION TITLE
Indexing issue: Failed creating fit_2048 (thumb: file hash is empty or too short ('')) ; and then converts every file it finds.

### DISCUSSION THREAD

**Discussion by NTragus** [photoprism/photoprism#3892] [Q&A]
Hello,

First of all, thanks for developping this app, it does exactly what I'm looking for!

I'm having an issue with indexing. It's the first time I'm indexing my library (~80,000 photos), and the index gets stuck every few hours with the following messages in the log (chronological order):
ERRO media: failed creating fit_2048 (thumb: file hash is empty or too short (''))
ERRO index: failed creating thumbnails for 'Image_xxx.png' (thumb: file hash is empty or too short (''))
ERRO index: failed adding main png file 'Image_xxx.png'

Once this happens, the indexing continues, but converts every file it founds without adding them. The following lines in the logs after the bug happens all look like:
INFO convert: converting 'Image_xxx.png' to 'Image_xxx.png.jpg'
This happens regardless of the format of the image, so I actually have lines stating "converting 'Image_xxx.jpg' to 'Image_xxx.jpg.jpg'"...
Docker logs do not show anything, and I could not find anything that helped solve the problem with photoprism's logs. 

Regarding the set-up, I'm on Raspberry pi 4B+ (4Go of memory, I followed the instructions for swap [here](https://docs.photoprism.app/getting-started/troubleshooting/docker/#raspbian)), and I access my files on a remote Seafile (localhost) mounted through Rclone on my disk under /mnt/seafile/. 

Any help would be much appreciated, as I have to reboot the Raspberry every few hours to keep indexing, and this is taking quite some time due to the size of the library.

Thanks a lot :)

**Comment by lastzero** [comment]
Since you are using a remote filesystem, it looks like there is an I/O bottleneck or lock preventing file fingerprints from being generated. You could try reducing the number of workers to 1 to see if that improves the situation. Also make sure that the file system is available at all times and that you have a stable network connection.

**Reply by NTragus** [reply]
Hello @lastzero,

Thanks for the answer, this sent me on the right direction, as I had never heard of fingerprinting before. I went through my Rclone configuration, only to find out the root cause to be linked to my Seafile set-up. Definitely not a photoprism issue :) 

I initially created this discussion since I thought it weird that Photoprism would start converting all files to .jpg, including .jpg files. What actually happens is Seafile crashes completely (though some data must be sent anyway for photoprism to still convert files? I did not delve too much into this bug).

As I had an old version of Seafile installed on bare metal, I spent the week-end upgrading to the latest docker installation of Seafile.
I'm currently running the sync again, with no issue so far.
Thanks for your fast response, I learnt something new that day!

**Reply by lastzero** [reply]
If you have large videos, try our latest development preview as it includes improvements to reduce I/O while indexing:
- https://github.com/photoprism/photoprism/pull/3893
